onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -676,6 +676,11 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                "which beam the 'k' th token came from for the 'j' th beam for batch 'i' in the current iteration",
                "M",
                OpSchema::Optional)
+        .Input(10,
+               "bias",
+               "Bias tensor with shape (hidden_size + hidden_size + v_hidden_size) from input projection",
+               "T",
+               OpSchema::Optional)
```

Comment:
you need to update ContribOperators.md and OperatorKernels.md to pass CI.

Reply:
fixed